web/src/components/Footer/index.tsx

```diff
@@ -20,7 +20,7 @@ import { DefaultFooter } from '@ant-design/pro-layout';
 
 export default () => (
   <DefaultFooter
-    copyright="2021 Apache APISIX"
+    copyright="2022 Apache APISIX"
```

Comment:
How about making it dynamic so that we don't have to update it every year?
```suggestion
    copyright=`${new Date().getFullYear()} Apache APISIX`
```